playwright/click.js

```diff
@@ -41,8 +41,9 @@ const context = await browser.newContext({
 });
 
 const page = await context.newPage();
+await page.goto("/");
 
-await page.locator("a[href='campfire-commerce/']").click();
+await page.getByText('Campfire Commerce').click();
```

Comment:
The locator doesn't work?

Reply:
Chromium runs into the same error:
```
locator.click: Error: strict mode violation: locator('a[href=\'campfire-commerce/\']') resolved to 2 elements:
    1) <a href="campfire-commerce/">Campfire Commerce</a> aka getByRole('link', { name: 'Campfire Commerce' })
    2) <a href="campfire-commerce/">…</a> aka getByRole('link').nth(1)
```suggestion
await page.getByText('Campfire Commerce').click();
```
```